example/build.sh

```diff
@@ -2,7 +2,7 @@
 
 rm -rf dist/ && mkdir -p dist/ &&
 babel server.js -o dist/server.js &&
```

Comment:
If we make this change, we can probably remove the installation of `GraphiQL` from example/package.json